make_prg/utils/io_utils.py

```diff
@@ -0,0 +1,221 @@
+import gzip
+import fileinput
+import shutil
+import uuid
+from Bio import AlignIO
+import os
+from loguru import logger
+from make_prg.from_msa import MSA
+from typing import Dict, Optional, List, Tuple
+from collections import defaultdict
+from pathlib import Path
+from zipfile import ZipFile
+
+
+def load_alignment_file(msa_file: [str, Path], alignment_format: str) -> MSA:
```

Comment:
```suggestion
def load_alignment_file(msa_file: Union[str, Path], alignment_format: str) -> MSA:
```

I guess you'll need to add `Union` from `typing`

Reply:
Yep, thanks for the correction!